inc/formanswer.class.php

```diff
@@ -170,32 +172,36 @@ public function canViewItem() {
       $ticket_user_table = Ticket_User::getTable();
       $item_ticket_table = Item_Ticket::getTable();
       $request = [
-         'SELECT' => Ticket_User::getTableField(User::getForeignKeyField()),
+         'SELECT' => [
+            Ticket_User::getTableField(User::getForeignKeyField()),
+            Ticket::getTableField('id'),
```

Comment:
Ticket ID added for debug needs, not a big issue to keep it in the SELECT clause